recipe/meta.yaml

```
@@ -31,13 +31,13 @@ requirements:
     - python
     - pip
     - setuptools
-    - llvmlite 0.33.0
+    - llvmlite 0.34.0
     - numpy
     - tbb-devel >=2019.5  # [linux]
 
   run:
     - python
-    - llvmlite >=0.33.0,<0.34
+    - llvmlite >=0.34.0,<0.35
     - {{ pin_compatible('numpy') }}
```

Shouldn't we use
```suggestion
    - {{ pin_compatible('numpy') }}
    # Working around an issue with NumPy 1.19.0
    # xref: https://github.com/numba/numba/issues/5929
    - numpy !=1.19.0
```
instead of adding the restriction to `test/requires` in https://github.com/conda-forge/numba-feedstock/pull/56/files#r470998911?

---

It seems it is just one niche case where it fails ( https://github.com/conda-forge/numba-feedstock/pull/56#issuecomment-674225286 ). So this seems too restrictive given that.